src/Fable.Transforms/Replacements.fs

```diff
@@ -2496,7 +2496,10 @@ let parseNum (com: ICompiler) (ctx: Context) r t (i: CallInfo) (thisArg: Expr op
     | "Pow", _ ->
         Helper.GlobalCall("Math", t, args, i.SignatureArgTypes, memb = "pow", ?loc = r)
         |> Some
-    | "ToString", [ ExprTypeAs(String, format) ] ->
+    | "ToString", [ ExprTypeAs(String, format) ]
```

Comment:
@MangelMaxime Perhaps add another case here for string const formats so we can issue warnings for unsupported formats, something like this:
```fs
| "ToString", [ StringConst format ] ->
```

Reply:
This is similar to the TODO, I put below.

I will have a look at implementing it, I though it would be difficult to do but in theory we just need a regex that match against a single char followed by some digits.

This will also solve the issue from your other comment, as we will be able to capture the "R" format at compile time.